packages/web-app-files/src/mixins/spaces/actions/editDescription.js

```diff
@@ -0,0 +1,63 @@
+import { mapActions } from 'vuex'
```

Comment:
As discussed, the edit description action is WIP. Could you add a comment in the code that this is not supposed to work, yet? Also please stick to the logic with the `resources` array and the name of the trigger method.